xbmc/video/windows/GUIWindowVideoNav.h

```diff
@@ -9,6 +9,7 @@
 #pragma once
 
 #include "GUIWindowVideoBase.h"
+#include "filesystem/VideoDatabaseDirectory/DirectoryNode.h"
```

Comment:
I think this include is not needed. `XFILE::VIDEODATABASEDIRECTORY::NodeType` is an `enum class` and can be forward declared.